### DISCUSSION TITLE
Autarkie Anzeige im Minus

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3623] [Q&A] [answered]
hallo Georg
hatte gestern folgende seltsame Anzeige im Diagramm Autarkie
auffallend, dass hier die Anzeige nicht farbfüllend dargestellt wird wie an normalen Tagen
![grafik](https://github.com/user-attachments/assets/f1146fbd-8ab0-4d20-99a7-7d5ce82380b3)
![grafik](https://github.com/user-attachments/assets/e91899d3-1ffe-4ae2-ae70-eab9388484f6)

**Comment by ledermann** [comment]
Welche Version?

**Reply by Zeppelin-PV** [reply]
Sowohl pr-8388 als auch 17.1.68

**Comment by gereons** [comment]
Kann ich bei mir mit dem pr-3533-Image auch reproduzieren, sogar für relativ viele Tage in der jüngeren Vergangenheit. 

![Bildschirmfoto 2024-11-04 um 14 32 17](https://github.com/user-attachments/assets/e2223406-396b-4c8d-bcd5-0682c0b4c7e9)

Wie man sieht sind die Werte für die Autarkie ganz knapp im Minus, ich tippe mal wegen Rundungsfehlern.

**Reply by Zeppelin-PV** [reply]
natürlich ist es bei mir auch das pr-3533, wie komm ich nur auf 8388 ?
und die develop 17.1.68
... auch bei mir sind es noch ein paar Tage mehr

**Comment by ledermann** [comment, marked as answer]
Ok, habe jetzt auch bei mir so einen Tag gefunden:
https://demo.solectrus.de/autarky/2024-01-31

Sieht auch für mich nach einem Rundungsfehler aus - #3625 ist angelegt.
